### DISCUSSION TITLE
Support for Android device to stream to Airplay speakers?

### DISCUSSION THREAD

**Discussion by artusmode** [philippe44/AirConnect#430] [Q&A] [answered]
Sorry if this is obvious to others, but i cannot seem to find a explanation or a solution to the following problem. I use Airconnect to create virtual airplay devices for my existing chromecast audio devices, works great.  These days, i have less and less speakers and devices that support Chromecast natively, but there are many more things like outdoor and speakers that support airplay. I am looking for a way to natively (without an app) from android stream to a airplay speaker. Any ideas if this is possible via Airconnect (or could be) or in general using something in the middle?  What am i missing?

**Comment by philippe44** [comment, marked as answer]
It's not the purpose of AirCast and unfortunately Google has made things such a way that it is almost impossible to create a virtual CC player and then redirect the audio somewhere.

**Reply by artusmode** [reply]
Understood, I figured this was the case but wanted to ask. Thanks!

**Comment by philippe44** [comment]
If you only need Spotify, you can use my "SpotConnect" application
